src/app/services/xml-parsers/msdesc-parser.ts

```diff
@@ -1004,15 +1004,44 @@ export class MsPartParser extends MsFragParser implements Parser<XMLElement> {
     }
 }
 
+let contMsDesc = 0;
```

Comment:
Move it inside MsDescParser as private property and improve the name in msDescCounter